lib/service/service.go

```diff
@@ -2952,6 +2954,7 @@ type proxyListeners struct {
 	mux              *multiplexer.Mux
 	tls              *multiplexer.WebListener
 	ssh              net.Listener
+	sshGrpc          net.Listener
```

Comment:
nit:

```suggestion
	sshGRPC          net.Listener
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I might also add a bit more info here. Something like:

```
// sshGRPC listens for traffic intended for the proxy.
// The SSH traffic is encapsulated in a GRPC stream to
// avoid the cost of an additional SSH handshake.
```